### ISSUE TITLE
Xdebug remote debugging broken over CLI with symfony 4.3 and DDtrace >= 0.28.0

### ISSUE THREAD

**Issue by andythorne** [DataDog/dd-trace-php#498]
Running symfony 4.3, xdebug no longer communicates with phpstorm after installing `dd-trace-php` >= 0.28.0. Any breakpoints in PHPStorm are ignored, and the xdebug log displays nothing.

PHP Setup:
```
PHP 7.1.30-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: May 31 2019 11:43:14) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.1.30-1+ubuntu16.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.7.1, Copyright (c) 2002-2019, by Derick Rethans
```

DDtrace installed with:
```
mkdir -p /tmp/datadog && \
    curl -L https://github.com/DataDog/dd-trace-php/releases/download/0.29.0/datadog-php-tracer_0.29.0_amd64.deb > /tmp/datadog/tracer.deb && \
    dpkg -i /tmp/datadog/tracer.deb
```

Reproducable by:
1. Set a breakpoint at line 1 of `bin/console`
2. run `./bin/console list`
3. Nothing happens

I suspect this is not just restricted to symfony4.3, as it fails to even hit line 1 breakpoint, before the symfony kernel has booted.

**Comment by SammyK** [comment]
Thanks for reporting this issue @andythorne! I was able to replicate this issue on my end. It appears that the changes from #477 prevent xdebug from overriding `zend_execute_ex()`. We are currently debugging this section of the tracer for an unrelated issue so we have this on our radar. 

Thanks again for the detailed report. We'll update this issue when we roll out a release that contains a fix. :)

**Comment by morrisonlevi** [comment]
I think #519, which is included in v0.30.0, should fix this issue by removing the `zend_execute_ex` on ddtrace's side. Can you upgrade and verify?

**Comment by labbati** [comment]
Hi @andythorne, I am closing this issue as we confirmed with other users that 0.30.0 fixed this bug. Please feel free to reopen it if this not true for you.
